cache/fast_lru_cache.cc

```diff
@@ -421,6 +519,9 @@ LRUCache::LRUCache(size_t capacity, size_t estimated_value_size,
                    int num_shard_bits, bool strict_capacity_limit,
                    CacheMetadataChargePolicy metadata_charge_policy)
     : ShardedCache(capacity, num_shard_bits, strict_capacity_limit) {
+  // // We must enforce strict capacity requirement, or else insertions
```

Comment:
This `strict_capacity_limit` refers to total charge, not total number of entries. This table implementation does need a strict capacity limit on number of entries, but not on total charge.

Reply:
Right. Perhaps it'd be good to keep a charge capacity limit flag, to maintain the load factor below the desired threshold. The meaning of ``strict_capacity_limit == false`` would change, though, to "overload the table, but only up to table capacity".